lib/src/log/log_data.dart

```diff
@@ -12,9 +12,52 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import 'log_fields.dart';
+
 /// LogData is data associated with a [Span]. Every LogData instance should
 /// specify at least one of Event and/or Payload.
 class LogData {
+  ///Returns a LogData instance.
+  LogData(
+    String event, {
+    DateTime timestamp,
+    String errorKind,
+    dynamic errorObject,
+    String message,
+    StackTrace stackTrace,
+    Map<String, dynamic> fields: const {},
+    this.payload,
```

Comment:
I would prefer if we removed `payload` entirely, since the `fields` parameter does the same job better.